src/components/performances/PerformanceDialog.jsx

```diff
@@ -8,21 +8,36 @@ import PerformanceForm from './PerformanceForm';
 import DFDialog from '../interface/dialog/DFDialog';
 import DialogHeader from '../interface/dialog/DialogHeader';
 import styles from '../styles';
+import addData from '../../firebase/utils/addData';
 
 class PerformanceDialog extends React.Component {
-  state = {
-    open: false
+  constructor(props) {
+    super(props);
+    this.state = { open: false };
+  }
+
+  handleClose = () => {
+    this.setState({ open: false });
   };
 
+
+  handleDup = async () => {
+    let vals = this.props.currentValues;
```

Comment:
this doesn't need to be a const, also we can destructure here.

const {
    currentValues: vals
} = this.props